src/components/forms/CreateRiddleForm.tsx

```diff
@@ -0,0 +1,45 @@
+import { FormContainer, useForm } from 'react-hook-form-mui';
+import { useCallback } from 'react';
+import { Button, Stack } from '@mui/material';
+import { Typography } from '@mui/material/styles/createTypography';
+
+import type { RiddleUpsertDetail } from '../../utils/Types';
+
+import { TextFieldFormComponent } from './generic/TextFieldFormComponent';
+import { AutocompleteLanguages } from './AutocompleteLanguages.tsx';
+import { AutocompleteDifficulties } from './AutocompleteDifficulties.tsx';
+import { AutocompleteUsers } from './AutocompleteUsers.tsx';
+import { RadioButtonFormComponent } from './generic/RadioButtonFormComponent.tsx';
+import { RadioButtonFormComponentBroad } from './generic/RadioButtonFormComponentBroad.tsx';
+
+export const CreateRiddleForm = () => {
```

Comment:
Musíme se dohodnout, jak budeme řešit to, že formulář se skládá z několika částí, abychom nemuseli každé části zvlášť vytvářet separátní url (stav, předávání dat atd.)

Reply:
Co třeba toto https://mui.com/material-ui/react-stepper/?

Reply:
Nevím, jak to přesně handluje stav, ale vypadá to dobře